clients/pkg/promtail/server/ui/assets_vfsdata.go

```diff
@@ -1,5 +1,6 @@
 // Code generated by vfsgen; DO NOT EDIT.
 
+//go:build !dev
```

Comment:
double checking that this is necessary

Reply:
Looks like it's just using a better (newer?) form of [build constraint](https://pkg.go.dev/cmd/go#hdr-Build_constraints) compared to the line below